psi4/driver/driver.py

```diff
@@ -130,6 +130,56 @@ def _energy_is_invariant(gradient, stationary_criterion=1.e-2):
     return translations_projection_sound, rotations_projection_sound
 
 
+def _process_displacement(derivfunc, method, molecule, displacement, n, ndisp, **kwargs):
+    """A helper function to perform all processing for an individual finite
+       difference computation.
+
+       Parameters
+       ----------
+       derivfunc : func
+           The function computing the target derivative.
+       method : str
+          A string specifying the method to be used for the computation.
+       molecule: psi4.core.molecule or qcdb.molecule
+          The molecule for the computation. Geometry setting is handled internally.
+       displacement : dict
+          A dictionary containing the necessary information for the displacement.
+          See driver_findif/_geom_generator.py docstring for details.
+       n : int
+          The number of the displacement being computed, for print purposes.
+       ndisp : int
+           The total number of geometries, for print purposes.
+
+       Returns
+       -------
+       wfn: psi4.core.wfn
```

Comment:
If you spec classes like the below in docstrings, they'll show up [linked in the manual](http://psicode.org/psi4manual/master/api/psi4.driver.vibanal_wfn.html). Not required, but it is good to spell them out and capitalize, like `psi4.core.Wavefunction`.

```
:py:class:`~psi4.core.Molecule`"
```

Reply:
Is the quote mark supposed to be there?

Reply:
no